src/sagemaker/mxnet/README.rst

```diff
@@ -31,13 +31,68 @@ In the following sections, we'll discuss how to prepare a training script for ex
 Preparing the MXNet training script
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-+-------------------------------------------------------------------------------------------------------------------------------+
-| WARNING                                                                                                                       |
-+===============================================================================================================================+
-| This required structure for training scripts will be deprecated with the next major release of MXNet images.                  |
-| The ``train`` function will no longer be required; instead the training script must be able to be run as a standalone script. |
-| For more information, see `"Updating your MXNet training script" <#updating-your-mxnet-training-script>`__.                   |
-+-------------------------------------------------------------------------------------------------------------------------------+
++--------------------------------------------------------------------------------------------------------------------------------------------------+
+| WARNING                                                                                                                                          |
++==================================================================================================================================================+
+| The structure for training scripts changed with MXNet version 1.3.                                                                               |
+| Please make sure you are referring to the correct section of this README when preparing your script.                                             |
```

Comment:
"Make sure you refer to the correct section of this README when you prepare your script."